tests/integration/golang/mlflow/artifact/list_s3_test.go

```diff
@@ -41,9 +42,20 @@ func (s *ListArtifactS3TestSuite) SetupTest() {
 	s3Client, err := helpers.NewS3Client(helpers.GetS3EndpointUri())
 	assert.Nil(s.T(), err)
 
+	createBuckets(s3Client)
 	s.s3Client = s3Client
 }
 
+func createBuckets(s3Client *s3.Client) {
+	for _, bucket := range []string{"bucket1", "bucket2", "bucket3"} {
+		// ignore error if bucket already exists.
+		_, _ = s3Client.CreateBucket(context.Background(), &s3.CreateBucketInput{
```

Comment:
once we delete the buckets during the tear down, we can actually capture the error and have a more robust setup

Reply:
done!